src/ripple/app/misc/impl/Manifest.cpp

```diff
@@ -193,6 +192,18 @@ ManifestCache::applicable (
         return ManifestDisposition::untrusted;
     }
 
+    return applyTrustedManifest (std::move(m));
```

Comment:
We were already using move semantics with `applyManifests` to allow the manifest to be moved all the way on to the cache.
`applyTrustedManifest` is just an added step in that flow.